cherrypicks.go

```diff
@@ -74,6 +76,51 @@ func getLatestReleaseFromApi(url string) ([]string, error) {
 	return v.Lts, nil
 }
 
+func getReleaseBranchesForCherryPick(
+	log *logrus.Entry,
+	pr *github.PullRequestEvent,
+	conf *config,
+	state *git.State,
+) ([]string, error) {
+
+	releaseBranches := []string{}
+
+	// fetch all the branches
+	err := git.Command("fetch", "github").With(state).Run()
+	if err != nil {
+		return releaseBranches, err
+	}
+
+	// get list of release versions
+	versions, err := getLatestReleaseFromApi(versionsUrl)
+	if err != nil {
+		return releaseBranches, err
+	}
+
+	repo := pr.GetRepo().GetName()
+	for _, version := range versions {
+		releaseBranch, err := getServiceRevisionFromIntegration(repo, "origin/"+version, conf)
```

Comment:
Just double-checking; we're still supposed to use `release_tool.py` here, right?

Reply:
Yes, for the Client LTS "bundle" we still don't have a replacement for the version mapping and there we use the release tool. After the Gateway and tools releases, it will be time to modernize the client release too :sparkles: